app/views/games/show.html.erb

```diff
@@ -31,29 +31,21 @@
   };
 
   $(function() {
-    $('.chessboard').each(function() {
-      var $chessboard = $(this);
-      var piecesURL = $chessboard.data('pieces-url');
 
-      // populates boards with pieces
+    setupBoard();
+
+    function setupBoard() {
+      $('.piece').remove();
+      var piecesURL = $('.chessboard').data('pieces-url');
+
       $.get(piecesURL).success(function(pieces) {
         pieces.forEach(function(piece, i){
-          if (!piece.is_captured) {
+          if (piece.row > -1 && piece.col > -1) {
```

Comment:
Captured pieces to side of chessboard. Let's add that later. I would get the basic JS functionality working to move the piece and remove - if applicable - the captured piece. We may not get to displaying the captured piece.